packages/mui-material/src/Modal/Modal.js

```diff
@@ -37,6 +37,7 @@ const ModalRoot = styled('div', {
 const ModalBackdrop = styled(Backdrop, {
   name: 'MuiModal',
   slot: 'Backdrop',
+  'aria-hidden': true,
```

Comment:
```suggestion
```

You need to add this where `ModalBackdrop` is used, what you are doing here doesn't make sense as the API is not supported.

Reply:
This should be updated to use the new suggested format now

Reply:
@mnajdova